base/src/main/java/org/mozilla/jss/provider/javax/crypto/JSSTrustManager.java

```diff
@@ -63,6 +65,15 @@ public class JSSTrustManager implements X509TrustManager {
 
     private String hostname;
     private boolean allowMissingExtendedKeyUsage = false;
+    private boolean enabledCertRevokeVerify = false;
+
+    public boolean isEnabledCertRevokeVerify() {
+        return enabledCertRevokeVerify;
+    }
+
+    public void setEnabledCertRevokeVerify(boolean enabledCertRevokeVerify) {
+        this.enabledCertRevokeVerify = enabledCertRevokeVerify;
+    }
```

Comment:
Could we replace `enabled` with `enable` so it's more consistent with `allowMissingExtendedKeyUsage`?